R/readGWLdata2.R

```diff
@@ -0,0 +1,120 @@
+# Copyright 2015 Province of British Columbia
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+# http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and limitations under the License.
+#'
+#' Read in groundwater data from files downloaded from GWL tool
```

Comment:
Added the comments. I thought to add an example and/or further instructions, but read it has to be executable on its own. I'm also unsure if this is something that you would want to be permanent since the data changes will potentially incur more significant change throughout the package.